packages/instant/src/components/ui/index.ts

```diff
@@ -1,4 +1,5 @@
 export { Text, TextProps, Title } from './text';
+export { Circle, CircleProps } from './circle';
```

Comment:
since we have a task to change this import syntax, moving forward would prefer to not  introduce more that will need to be switched out

Reply:
does it make sense to remain consistent until we make the switch?
```
src/components/amount_placeholder.tsx:import { Text } from './ui';
src/components/buy_button.tsx:import { Button, Text } from './ui';
src/components/buy_order_state_buttons.tsx:import { Button, Flex, Text } from './ui';
src/components/erc20_asset_amount_input.tsx:import { Container, Flex, Icon, Text } from './ui';
src/components/erc20_token_selector.tsx:import { Circle, Container, Flex, Text } from './ui';
src/components/instant_heading.tsx:import { Container, Flex, Icon, Spinner, Text } from './ui';
src/components/order_details.tsx:import { Container, Flex, Text } from './ui';
src/components/placing_order_button.tsx:import { Button, Container, Spinner, Text } from './ui';
src/components/scaling_input.tsx:import { Input } from './ui';
src/components/search_input.tsx:import { Container, Flex, Icon, Input, InputProps } from './ui';
src/components/secondary_button.tsx:import { Button, ButtonProps, Text } from './ui';
src/components/sliding_error.tsx:import { Container, Flex, Text } from './ui';
src/components/sliding_panel.tsx:import { Container, Flex, Icon, Text } from './ui';
src/components/zero_ex_instant_container.tsx:import { Container, Flex } from './ui';
src/components/zero_ex_instant_overlay.tsx:import { Overlay } from './ui';
```

Reply:
i'm cool w/ this